src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -19,13 +19,22 @@ public void actionPerformed(ActionEvent e) {
 		switch (e.getActionCommand()){
 			case "Figure Joe 1":
 				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-			break;
+				break;
 			case "Figure Joe 2":
 				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				break;
 			case "Figure Jane":
 				AbstractDriverAdapter driver = new AbstractDriverAdapter(driverManager.getCurrentDriver());
 				FiguresJane.figureScript(driver);
+				break;
+			case "Square":
+				AbstractDriverAdapter driverSquare = new AbstractDriverAdapter(driverManager.getCurrentDriver());
+				ComplexCommandFactory.square(driverSquare).execute();
+				break;
+			case "Hexagon":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.